webui/react/src/hooks/useModal/useModal.tsx

```diff
@@ -34,7 +34,7 @@ export enum ModalCloseReason {
 
 const DEFAULT_MODAL_PROPS: Partial<ModalFuncProps> = {
   maskClosable: true,
-  style: { minWidth: 600 },
+  style: { minWidth: 300 },
```

Comment:
**nit:** On chrome, for the device/responsive mode, the smallest screen size support is [Galaxy Fold](https://www.samsung.com/global/galaxy/galaxy-fold/) and it's `280`. If that breaks and makes some modal screens look bad it's fine. I was thinking also, 280 might be easier to find in the code than a "clean and even" 300 if we ever need to reference it.